docs/surveys/2020-jupyterlab-survey.md

```diff
@@ -247,25 +247,25 @@ and machine learning on your laptop/ desktop (non-cloud) for data science? (pick
 ### Scale
 
 14. How do you scale and schedule your workloads? (up to 4)
-    - Cluster - Jupyter Enterprise Gateway.
-    - Cluster - Kubernetes (or similar e.g. Mesos, Swarm, Slurm).
+    - 🖥️ They run just fine on my local machine.
+    - ❓I need to scale, but don't know how.
+    - Server - On premise HPC/ data center.
+    - Server - Cloud (e.g. AWS EC2).
+    - Service - ML/ AI (e.g. AWS SageMaker).
     - Cluster - Spark (or similar e.g. Hadoop, Dask).
-    - Elyra.
-    - Horovod.
+    - Cluster - Kubernetes (or similar e.g. Mesos, Swarm, Slurm).
+    - Cluster - Jupyter Enterprise Gateway.
     - Jupyter BinderHub.
-    - Kubeflow.
     - Quantum (e.g. D-Wave).
-    - Server - Cloud (e.g. AWS EC2).
-    - Server - On premise HPC/ data center.
-    - Service - Cloud Queries (e.g. AWS Presto, AWS Athena).
-    - Service - ML/ AI (e.g. AWS SageMaker).
-    - Workflow - Airflow.
-    - Workflow - Cloud service (e.g. AWS Batch).
-    - Workflow - Nextflow, WDL, CWL.
-    - Workflow - Papermill.
+    - Horovod.
+    - Kubeflow.
+    - Elyra.
     - Workflow - Snakemake.
-    - ❓I need to scale, but don't know how.
-    - 🖥️ They run just fine on my local machine.
+    - Workflow - Papermill.
```

Comment:
It probably makes sense to add Dask as it's own workflow item as well.

Reply:
@willingc I'll agree in splitting out Dask from Spark. Here, _workflow_ has a stricter definition of reproducible, parameterized, user-specified DAG-style tasks where it's less of a detailed script and more of a string of ordered tasks. So Prefect might make more sense. 
https://examples.dask.org/applications/prefect-etl.html

Reply:
Totally up to you @LayneSadler where to put Dask. It would be good to include somewhere even if not under the DAG workflow items.